### DISCUSSION TITLE
When attempting to download OsuLazer this is what pops up

### DISCUSSION THREAD

**Discussion by assinator1234** [ppy/osu#20931] [Q&A]
### Type

Other

### Bug description

When trying to do a install this error code pop's up. I have wiped the file's and tried again and it always pops up with the same error. I went into event viewer like I've seen in other forum posts and I saw nothing for the install(Still attached in case it was missed)

### Screenshots or videos

![Osulazer Error Popup](https://user-images.githubusercontent.com/68050782/197745927-34918309-b2d9-4adf-b9d0-945275fae8a2.png)
[EventViewer.zip](https://github.com/ppy/osu/files/9859279/EventViewer.zip)

### Version

2022.1022.0

### Logs

[Squirrel.log](https://github.com/ppy/osu/files/9859229/Squirrel.log)

**Comment by assinator1234** [comment]
> 

I found it in resource monitor and when I closed and reopened it didnt do anything just in case I checked my task manager and I found it will run in the background. Is there a reason that it would do that?
![image_2022-10-25_063429314](https://user-images.githubusercontent.com/68050782/197751596-53b3d595-1e98-4e90-ab64-75c0d058a4ce.png)

**Comment by assinator1234** [comment]
I reinstalled and now it wont open at all I still get the same error as the first one I posted idk why after the reinstallation it would entirely quit working

**Comment by bdach** [comment]
From the squirrel log:

```
2022-10-25 05:55:20.9222 [FATAL] - Program: Encountered fatal unhandled exception.: System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\chuck\AppData\Local\osulazer\app-2022.1022.0\osu!.exe' with working directory 'C:\Users\chuck\AppData\Local\osulazer\app-2022.1022.0'. The requested operation requires elevation.)
 ---> System.ComponentModel.Win32Exception (740): An error occurred trying to start process 'C:\Users\chuck\AppData\Local\osulazer\app-2022.1022.0\osu!.exe' with working directory 'C:\Users\chuck\AppData\Local\osulazer\app-2022.1022.0'. The requested operation requires elevation.
   at System.Diagnostics.Process.StartWithCreateProcess(ProcessStartInfo )
   at System.Diagnostics.Process.StartCore(ProcessStartInfo )
```

Installing lazer should not require administrator privileges. This suggests that there is a permissions issue with your `AppData\Local` folder.

**Reply by assinator1234** [reply]
how would I fix that, or diagnose whats wrong?

**Reply by peppy** [reply]
It could also mean that whoever is your system administrator has locked you out of installing things (ie. if this is a school or work PC).

**Reply by assinator1234** [reply]
its my own pc. but when I restart it the drive repairs for some reason if that could be the issue

**Reply by Ghost1928563945** [reply]
Did you try running as admin?

**Reply by assinator1234** [reply]
the download? also while looing at the folder it shows that it's read only, is that something that could be a problem?
